packages/components/stories/ActionBar.js

```diff
@@ -213,6 +213,40 @@ const basicProps = {
 	actions,
 	multiSelectActions,
 };
+const multiDelete = {
+	label: 'Delete',
+	icon: 'talend-trash',
+	onClick: action('multiple delete'),
+	bsStyle: 'icon-text',
+};
+const multiDuplicate = {
+	label: 'Duplicate',
+	icon: 'talend-files-o',
+	onClick: action('multiple duplicate'),
+	bsStyle: 'icon-text',
+};
+const multiUpdate = {
+	label: 'Update',
+	icon: 'talend-file-move',
+	onClick: action('multiple update'),
+	bsStyle: 'icon-text',
+	className: 'separated',
+};
+const multiFavorite = {
+	label: 'Favorite',
+	icon: 'talend-star',
+	onClick: action('multiple favorite'),
+	bsStyle: 'icon-text',
```

Comment:
this create prototypes warning in console.
bsStyle has a set of existing value related to bootstrap
so to use it prefere className: 'btn-icon-text'